web/modules/custom/rsvplist/src/Controller/ReportController.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\rsvplist\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\database\Connection;
+use Drupal\Core\Messenger\MessengerInterface;
+
+/**
+ * ReportController class to process the routes.
+ */
+class ReportController extends ControllerBase {
+
+  /**
+   * A database connection object.
+   *
+   * @var Drupal\Core\database\Connection
+   */
+  protected Connection $databaseConnection;
```

Comment:
As per coding standards, it is not necessary to explicitly mention the type just `protected $databaseConnection` will work.